src/lib/track.js

```diff
@@ -10,6 +11,7 @@ import type {
 
 export const track = <T>(config : Config, trackingType : TrackingType, trackingData : T) => {
     const encodeData = data => encodeURIComponent(btoa(JSON.stringify(data)));
+    const cartId = getOrCreateCartId().cartId;
```

Comment:
I think you want a nonExpired cartId here, so perhaps the `getOrCreateValidCartId`, or `resetExpiredCartId` is more relevant here?

Reply:
Honestly, I'd rather this didn't exist at all. The 'track' method shouldn't concern itself with this step in the first place. By allowing this to fail gracefully I think we're opening the door to people inadvertently botching the setup process without realizing it.